openpype/settings/entities/schemas/projects_schema/schema_project_photoshop.json

```diff
@@ -131,6 +131,19 @@
                         }
                     ]
                 },
+                {
+                    "type": "dict",
+                    "collapsible": true,
+                    "key": "CollectReview",
+                    "label": "Collect Review",
+                    "children": [
+                        {
+                            "type": "boolean",
+                            "key": "publish",
+                            "label": "Publish review"
+                        }
+                    ]
+                },
```

Comment:
Should `CollectReview` have just been added in the `template_publish_plugin` template data below. It'd allow the setting to either make the plug-in active or not and or optional?

Also please make sure that an instance that is collected with `publish=False` still runs fine if the user manually enables it. Because I believe InstancePlugin collectors would not run for it if that instance was `publish=False` from the beginning - and thus e.g. `CollectInstanceAnatomyData` would not run for that instance if the user enabled it after collecting.

Reply:
Collector can't be optional (user don't have ability to deactivate collectors).